test/integration/app.js

```diff
@@ -57,21 +57,22 @@ describe('app', () => {
   describe('existing GP page', () => {
     it('should return a GP Page for a valid Org Code', (done) => {
       chai.request(app)
-        .get(`${constants.SITE_ROOT}/43213`)
+        .get(`${constants.SITE_ROOT}/44125`)
```

Comment:
yeah agreed, I was a tad lazy here... @neilbmclaughlin has made some great improvements to the tests on 'gp-finder' repo. We should do similar here.